features/gezag-minderjarige/1.3-is-naar-buitenland-geemigreerd-geweest.feature

```diff
@@ -0,0 +1,94 @@
+#language: nl
+
+Functionaliteit: 1.3 - is naar buitenland geëmigreerd geweest
+  Precondities voor deze vraag:
+  | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                       |
+  | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                       |
+
+  Een minderjarige die naar het buitenland geëmigreerd geweest of tijdelijk woonachtig is geweest in het buitenland kan het gezag uit het
+  buitenland hebben meegenomen. In die gevallen wordt het gezag niet vastgelegd in BRP (ook niet in het Gezagsregister). Het gezag kan dus niet
+  betrouwbaar worden achterhaald.
+
+  Vanaf 1 mei 2011 zijn de regels van het Haags Kinderbeschermingsverdrag 1996 van toepassing.
+  Vanaf deze datum wordt het toepasselijk recht bepaald door het recht van het land van de gewone verblijfplaats van het kind.
+  Daarnaast blijft het verkregen gezag behouden na wijziging van de gewone verblijfplaats. Gezag verhuist dus mee met het kind.
+  
+  Hoe achterhalen:
+  Bepaal of de persoon als geboorteland Nederland heeft en datum vestiging in Nederland voor komt.
+  
+  Gebruikte velden:
+    - Geboorteland -> 01.03.30
+    - Datum vestiging in Nederland -> 08.14.20
+
+    Achtergrond:
+      Gegeven de persoon 'Ruth' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Maurice' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Ruth' en 'Maurice' zijn met elkaar gehuwd
+      En de persoon 'Lisa' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is in Nederland geboren
+      * is ingeschreven in de BRP
+      * heeft 'Ruth' als ouder 1
+      * heeft 'Maurice' als ouder 2
+
+  Regel: Kind is altijd in nederland gebleven
+
+    Scenario: Het kind is geboren in Nederland en verblijft in Nederland er is sprake van TweehoofdigOuderlijkGezag
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+      
+  Regel: Het kind is naar het buitenland geëmigreerd
+
+    Scenario: Het kind is geboren in Nederland en is geëmigreerd er is sprake van TweehoofdigOuderlijkGezag
```

Comment:
Deze regel en scenario was besproken in een dagstart. Dit klopt niet, omdat bij emigratie de gemeente van inschrijving op RNI wordt gezet. Dat betekent dat het niet verder komt dan vraag 1.1 en het uiteindelijk resultaat GezagNietTeBepalen is.

De bijbehorende step definitie klopt daardoor ook niet.

Reply:
Uitschrijven uit de BRP of inschrijven in de RNI door uit te schrijven bij de gemeente is pas verplicht als je langer dan 8 maanden in een jaar in het buitenland verblijft, je kan ook verhuizen zonder je (direct) uit te schrijven.

Het gezag over je kind verhuist met je mee

Reply:
De emigratie die je hier beschrijft kun je niet uit de BRP halen, want is niet geregistreerd. In maand 7 is het kind dus gewoon ingezetene van NL. Voor de BRP is deze persoon niet geëmigreerd. 
Dit scenario kan dus niet in de BRP.
Denk wel dat de leerplichtsambtenaar aan de bel trekt, en een onderzoek wordt gestart.....:-)

Reply:
@CathyDingemanse @FrozenSync snappen jullie waarom deze conversatie gesloten is (resolved)? het scenario staat er nog

Reply:
Ik denk dat de conversatie is gesloten op basis van eigen gelijk. Dat is een beetje onhandig tijdens een discussie/onduidelijkheid/onenigheid. Wellicht is het beter als degene die de vraag stelt, ook degene is die de conversatie sluit. Zo voorkom je dat conversaties voortijdig gesloten worden en minder zichtbaar raken.

Reply:
Scenario is verwijderd.